openai_dive/src/v1/resources/shared.rs

```diff
@@ -136,18 +136,32 @@ pub enum LastErrorCode {
 #[serde(rename_all = "snake_case")]
 pub enum FinishReason {
     /// API returned complete message, or a message terminated by one of the stop sequences provided via the stop parameter.
-    #[serde(rename = "stop")]
+    #[serde(rename = "stop", alias = "STOP")]
     StopSequenceReached,
     /// Incomplete model output due to max_tokens parameter or token limit.
-    #[serde(rename = "length")]
+    #[serde(rename = "length", alias = "MAX_TOKENS")]
     TokenLimitReached,
     /// Omitted content due to a flag from our content filters.
-    #[serde(rename = "content_filter")]
+    #[serde(
```

Comment:
![image](https://github.com/user-attachments/assets/4f909777-30b5-4377-987b-45bc6a0b8376)

Bundled all of these together just to limit having multiple enums to handle. According to [Gemini docs](https://cloud.google.com/vertex-ai/generative-ai/docs/reference/python/latest/vertexai.preview.generative_models.FinishReason?authuser=1) seems like they all go under the same category.